tests/ConversationTest.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+namespace RonasIT\Chat\Tests;
+
+use Illuminate\Support\Facades\Notification;
+use PHPUnit\Framework\Attributes\DataProvider;
+use RonasIT\Chat\Models\Conversation;
+use RonasIT\Chat\Notifications\ConversationDeletedNotification;
+use RonasIT\Chat\Tests\Models\User;
+use RonasIT\Chat\Tests\Support\ModelTestState;
+
+class ConversationTest extends TestCase
+{
+    protected static User $sender;
+    protected static User $recipient;
+    protected static User $someAuthUser;
+
+    protected static ModelTestState $conversationTestState;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        self::$sender ??= User::find(1);
+        self::$recipient ??= User::find(2);
+        self::$someAuthUser ??= User::find(3);
+
+        self::$conversationTestState = new ModelTestState(Conversation::class);
+    }
+
+    public function testGetBySender()
+    {
+        $response = $this->actingAs(self::$sender)->json('get', '/conversations/1');
+
+        $response->assertOk();
+
+        $this->assertEqualsFixture('get_conversation.json', $response->json());
```

Comment:
let's upgrade required LaravelHelpers package version to reduce of json extension

```suggestion
        $this->assertEqualsFixture('get_conversation', $response->json());
```